Makefile

```
@@ -66,12 +66,9 @@ coverage: ## check code coverage quickly with the default Python
 	$(BROWSER) htmlcov/index.html
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/hcloud.rst
-	rm -f docs/modules.rst
-	sphinx-apidoc -o docs/ hcloud
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
-	$(BROWSER) docs/_build/html/index.html
+	## $(BROWSER) docs/_build/html/index.html
```

```suggestion
     $(BROWSER) docs/_build/html/index.html
```

Re enable this before merge